packages/theme/build/writeDefaultValue.js

```
@@ -0,0 +1,103 @@
+
+      if (subVar.includes('calc(')) {
+        subVar = subVar.replace('calc(', '').replace(')', '').replace('px', '')
+        subVar = eval(subVar) + 'px'
+      }
+
+      return subVar
+    }
+    return defaultValue
+  }
+
+  // 写入文件
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)